pyproject.toml

```diff
@@ -184,28 +184,71 @@ exclude_lines = [
 ]
 
 # Configure linters
-# ruff
+
+[tool.codespell]
+ignore-words-list = "emac,hist,oce,pres,vas,runn"
+
 [tool.ruff]
 fix = true
 line-length = 79
 show-fixes = true
+
 [tool.ruff.lint]
 select = ["ALL"]
+# syntax from github.com/astral-sh/ruff/issues/10878
+exclude = ["archive/*"]
 ignore = [
     # TODO: when replacing `prospector` with `ruff`, ignore rules that are
     # overly technical or conflict with other rules.
-    "E501",  # Disable line-too-long as this is taken care of by the formatter.
-    "D105",  # Disable Missing docstring in magic method as these are well defined.
+    "E501",     # Disable line-too-long as this is taken care of by the formatter.
+    "D105",     # Disable Missing docstring in magic method as these are well defined.
+    # from ESMValTool: Ignore until someone has time to fix this:
+    "B011",     # Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`
+    "UP038",    # Use `X | Y` in `isinstance` call instead of `(X, Y)`
```

Comment:
Would it be OK with you to just fix these issues? It's only 4 of them anyway.

Reply:
done in 7f6218ed7381988da1166f042fad7e32632d80ac Monsieur Le Bouwe :beers: